packages/opc-back-to-top/package.json

```diff
@@ -0,0 +1,47 @@
+{
+  "name": "@one-platform/opc-back-to-top",
+  "version": "0.0.1-prerelease",
+  "description": "Back to Top Component which enables smooth navigation from bottom scroll bar.",
```

Comment:
rename to `Back to Top component that enables smooth scroll from the bottom of the page to the top of the page`